examples/cli.rs

```diff
@@ -422,7 +423,29 @@ async fn run<C: Store + MessageStore>(subcommand: Cmd, config_store: C) -> anyho
         Cmd::Block => unimplemented!(),
         Cmd::Unblock => unimplemented!(),
         Cmd::UpdateContact => unimplemented!(),
-        Cmd::ListGroups => unimplemented!(),
+        Cmd::ListGroups => {
+            let manager = Manager::load_registered(config_store)?;
+            let mut groups = manager.get_groups()?.peekable();
+            if groups.peek().is_some() {
+                println!("Groups: ");
+                for group in groups {
+                    match group {
+                        Ok(group) => {
+                            let title = String::from_utf8(group.title.clone())?;
+                            let key = base64::encode(group.public_key.clone());
+                            println!("{:?} {:?}",key,  title);
+                        }
+                        Err(e) => {
+                            println!("Error: {:?}", e);
```

Comment:
Maybe use the `error!` macro here instead? And avoid using the `Debug` representation of the error as well.

Reply:
Actually i would love to use `log::Error` and `log::Info` or `error!` but they don't print anything to my console, so I ended up with `println!` statements. Regarding the `{:?}`, I still have to read some basics about the representations.

Reply:
the macro `error!` isn't defined and i changed it to an error representation.